pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -21,6 +21,13 @@ let
   buildKernel = any (n: n == configFile) [ "kernel" "all" ];
   buildUser = any (n: n == configFile) [ "user" "all" ];
 
+  # remove this patch at the next ZFS release
+  reapplyPathSanitizerPatch = fetchpatch {
+    url =
+      "https://github.com/openzfs/zfs/commit/03f036cbccdd8699f5fe8540ef317595a35bceb8.patch";
```

Comment:
```suggestion
    url = "https://github.com/openzfs/zfs/commit/03f036cbccdd8699f5fe8540ef317595a35bceb8.patch";
```

Reply:
Is there any code-formatting styleguide I can follow? I used https://github.com/nix-community/nixpkgs-fmt to format, as I thought this is used in the code base, but it completely reformatted the document (in particular the arguments in the top level function).